pyracf/common/logger.py

```diff
@@ -93,33 +95,65 @@ def log_debug(self, header_message: str, message: str) -> None:
         )
         print(f"{header}\n{message}")
 
-    def redact_strings(
+    def __redact_request_dictionary(
         self,
-        string: Union[str, bytes],
-        redact_strings: List[str],
-    ) -> str:
-        """Redact a list of strings or sequences of bytes in a string or bytes object"""
-        if not redact_strings:
-            return string
-        for to_redact in redact_strings:
-            if isinstance(string, bytes):
-                string = self.redact_string(
-                    string, redact_string=bytes(to_redact, "utf-8")
-                )
-            else:
-                string = self.redact_string(string, redact_string=to_redact)
-        return string
+        dictionary: dict,
+        secret_traits: dict,
+    ) -> dict:
+        """Redact a list of specified secret traits in a request dictionary"""
+        for seg_trait in secret_traits:
```

Comment:
This is small, but I would just call this variable `trait` instead of `seg_trait`.